server/server.js

```diff
@@ -20,6 +20,14 @@ app.get(
   passport.authenticate("github", { scope: ["user:email"] })
 );
 
+app.get(
+  "/auth/github/callback",
+  passport.authenticate("github", { failureRedirect: "/login" }),
```

Comment:
Where should failure be redirected? We don't have `/login` url.

Reply:
I've updated redirection routes, failure to "/" and success to "/trainee"